src/v2/Apps/getAppRoutes.tsx

```diff
@@ -1,6 +1,7 @@
 import { buildAppRoutes } from "v2/Artsy/Router/buildAppRoutes"
 import { RouteConfig } from "found"
 import { routes as artistRoutes } from "v2/Apps/Artist/routes"
+import { routes as artistsRoutes } from "v2/Apps/Artists/routes"
```

Comment:
Might as well just rename file to `artistsRoutes.tsx`. Been meaning to do that for other route files